tests/data/group_system/group_software/group_gnome/group_gnome_screen_locking/rule_dconf_gnome_screensaver_lock_enabled/comment.fail.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+
+yum -y install dconf
+
+# It is ok if string is not found in any file
+file=$(grep -R "lock-enabled" /etc/dconf/db/local.d) || true
```

Comment:
nack, we need to test using acutal dconf, not checking files. let's do real integration tests here. Look up dconf read